test/appTest/controllersTest/signUpTest.js

```diff
@@ -0,0 +1,110 @@
+const chai = require('chai'),
+  app = require('./../../../app'),
+  expect = chai.expect,
+  dictum = require('dictum.js');
+
+describe('sign up with all correct data', () => {
+  it('When all the data is correct we will get a message with the new user`s name', done => {
+    const user = {
+      name: 'name',
+      lastName: 'lname',
+      email: 'name@wolox.co',
+      password: '12345qwe',
+      role: 'Admin'
+    };
+
+    chai
+      .request(app)
+      .post('/user')
+      .send(user)
+      .then(res => {
+        expect(res.text).to.equal(`A new user named: ${user.name}, has been created`);
+        expect(res).to.have.status(201);
+        dictum.chai(
+          res,
+          'When all the data is correct, we will get a message with the new user`s name, as a response'
+        );
+        done();
+      });
+  });
+});
+
+describe('sign up without an email', () => {
+  it('When someone tries to sign up without an email, a warning message will apear and the request wont be resolved', done => {
+    const user = {
+      name: 'name',
+      lastName: 'lname',
+      email: '',
+      password: '12345qwe',
+      role: 'Admin'
+    };
+    chai
+      .request(app)
+      .post('/user')
+      .send(user)
+      .catch(error => {
+        expect(error.toString()).to.equal('Error: Bad Request');
```

Comment:
Why to string ?

Reply:
Im getting the error as expected, but for some reason, is not recognizing the message inside of it, but returning some kind of object 

I could try expecting error.message

Reply:
use error.message